src/features/query-bar/validator.js

```diff
@@ -1,15 +1,19 @@
 // @flow
+import { isInteger, isNaN } from 'lodash';
```

Comment:
Please do not import the root lodash module directly, as it's very likely to increase our bundle size. You can import single functions via `import isInteger from 'lodash/isinteger';`